components/Blocks/PhotoHero.css

```diff
@@ -0,0 +1,46 @@
+.root {
+  width: 100%;
+  padding: 5em 0;
+  position: relative;
+  background-size: cover;
+  color: var(--color-white);
```

Comment:
We would, it’s just a case of variable names being reusable *and* explicit not sitting well with me. Variables named after their function and not their style just makes more sense. Totally trivial though so feel free to disregard and merge.